app/forms/contact/create_chair_of_governors_contact_form.rb

```diff
@@ -0,0 +1,24 @@
+class Contact::CreateChairOfGovernorsContactForm < Contact::CreateProjectContactForm
+  attribute :category, default: "school_or_academy"
```

Comment:
I don't think we need this? It's declated on the base form and we hard wire it's value in the `save` method.